code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -19,7 +19,7 @@
 	name = "beanbag slug"
 	icon_state = "pellet"
 	damage = 10
-	stamina = 55
+	stamina = 138
```

Comment:
This still wont make them usable as people will still start regenerating stam right away, so you would have to keep shooting them while they were downed in order to cuff

Reply:
up'd to 145 which with two shots does 290 stamina damage meaning they should be down for 5 seconds? if Im not mistaken since regen is 15 stamina a tick 2 seconds.